docs/coding-guidelines/javascript.md

```diff
@@ -302,32 +305,37 @@ Multi-line comments that are not a jsdoc comment should use `//`:
 
 ## Equality
 
-Strict equality checks (===) must be used in favor of abstract equality checks (==). The only exception is when checking for both undefined and null by way of null.
+Strict equality checks (===) must be used in favor of abstract equality checks (==). The only exception is when checking for both undefined and null by way of null, though it is preferable to use Lodash's [`isNil`](https://lodash.com/docs#isNil) for this purpose.
 
 ```js
-// Check for both undefined and null values, for some important reason.
-if ( undefOrNull == null ) {
-	// Expressions
+// Check that 'someValue' is either undefined or null, for some important reason.
+
+// Good
+if ( someValue == null ) {
+	...
+}
+
+// better
+if ( isNil( someValue ) ) {
+	...
 }
 ```
 
 ## Type Checks
 
-These are the preferred ways of checking the type of an object:
+When checking the type of a value, use one of the following utilities from [Lodash](https://lodash.com/):
```

Comment:
I'd leave this line untouched, as the following examples speak for themselves.

Reply:
I agree, I'll revert but instead of referring to the type of an `object` I'll use value. I think object is a little misleading in the existing text since we could be checking any type or variable.